jdaviz/configs/specviz/helper.py

```diff
@@ -189,20 +189,26 @@ def flip_y(self):
     def show(self):
         self.app
 
-    def set_redshift_slider_bounds(self, lower=None, upper=None, step=None):
+    def set_redshift_slider_bounds(self, range=None, step=None):
         '''
-        Set the upper, lower, or both bounds of the redshift slider. Note
-        that this does not do any sanity checks on the numbers provided based
-        on whether the slider is set to Redshift or Radial Velocity.
+        Set the range and/or step of the redshift slider. Set either/both to 'auto'
+        to default based on the limits of the spectrum plot.
+
+        Parameters
+        ----------
+        range
```

Comment:
What if default (`None`) is used -- what is the behavior then? Is the given range in the form of `(min, max)` and is it inclusive or exclusive or inclusive on min only like Python?
```suggestion
        range : tuple of float or `None`
```

Reply:
see 889e2d4: the range must be a float, None, or 'auto'.  If None, the current value remains unchanged.